crates/matrix-sdk/tests/integration/room/joined.rs

```diff
@@ -748,3 +749,47 @@ async fn subscribe_to_typing_notifications() {
     join_handle.await.unwrap();
     assert_eq!(typing_sequences.lock().unwrap().to_vec(), asserted_typing_sequences);
 }
+
+#[async_test]
+async fn get_suggested_user_role() {
+    let (client, server) = logged_in_client_with_server().await;
+
+    mock_sync(&server, &*test_json::DEFAULT_SYNC_SUMMARY, None).await;
+
+    let sync_settings = SyncSettings::new().timeout(Duration::from_millis(3000));
+    let _response = client.sync_once(sync_settings).await.unwrap();
+    let room = client.get_room(&DEFAULT_TEST_ROOM_ID).unwrap();
+
+    let role_admin =
+        room.get_suggested_user_role(&UserId::parse("@example:localhost").unwrap()).await.unwrap();
```

Comment:
Here and below, there's a ruma shortcut for parsing the user id:

```suggestion
        room.get_suggested_user_role(user_id!("@example:localhost")).await.unwrap();
```